pkg/cmd/pipeline/start.go

```diff
@@ -197,6 +207,31 @@ func mergeParam(pr *v1alpha1.PipelineRun, optPar []string) error {
 	return nil
 }
 
+func mergeSvc(pr *v1alpha1.PipelineRun, optSvc []string) error {
+	svcs, err := parseSvc(optSvc)
+	if err != nil {
+		return err
+	}
+
+	if len(svcs) == 0 {
+		return nil
+	}
+
+	for i := range pr.Spec.ServiceAccounts {
+		if v, ok := svcs[pr.Spec.ServiceAccounts[i].TaskName]; ok {
+			pr.Spec.ServiceAccounts[i] = v
+			delete(svcs, v.TaskName)
```

Comment:
wondering do we really need this delete call?

Reply:
Yes, because I use the remaining args to append if a user has given some passed some new args.